modules/enquote.py

```diff
@@ -0,0 +1,10 @@
+from .base import Module
+	
+class ENQUOTE(Module):
+    DESCRIPTION = "writes quote en quote however many times"
+    ARGC = 1
+    def response(self, query, message):
+	if(query < 0 or query > 10):

```

Comment:
Sorry but I... don't really get what this module is supposed to be for?

Reply:
yeah, it's a pretty dumb meme that I have, doesnt really make sense to add it Im gonna delete it